meson.build

```diff
@@ -0,0 +1,34 @@
+
+project('ashuffle', 'c', version: '1.0.3', default_options: ['c_std=c99'])
+
+add_global_arguments([
+	'-Wextra',
+	'-pedantic' ],
+	language: 'c'
+)
+
+sources = [
+	'src/args.c',
+	'src/ashuffle.c',
+	'src/getpass.c',
+	'src/list.c',
+	'src/rule.c',
+	'src/shuffle.c'
+]
+
+clientlib = dependency('libmpdclient')
+
+gdb = find_program('gdb', required: false)
+
+
```

Comment:
It would be nice to have more uniform spacing here. Let’s say one space?

Reply:
gdb stuff was removed anyway